tensorboard/webapp/metrics/views/card_renderer/image_card_component.ng.html

```diff
@@ -79,7 +79,7 @@
     <img
       alt="Image at step {{ stepValues[stepIndex] }}"
       src="{{ imageUrl }}"
-      [style.filter]="cssFilter()"
+      [ngStyle]="{'filter': cssFilter()}"
```

Comment:
Angular complains on dynamic bindings to style.filter, but it is okay with ngStyle.
`'WARNING: sanitizing unsafe style value contrast(100%) brightness(2) (see http://g.co/ng/security#xss).`